### DISCUSSION TITLE
[预览] v2.8.9

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#4673] [分享 / Share]
包含 [v2.8.9](https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.8.9) 的所有更新内容.

✨新增
- `下载视频 - WASM 混流输出` 支持更持久的缓存方式. (PR #4667 by [WakelessSloth56](https://github.com/WakelessSloth56))
- `外置稍后再看` 和 `启用快速收藏` 增加了 `显示方式` 选项, 出现无法自动适配的排版问题时可以尝试手动调整此设置. (#4532)
- 新增插件 `动态过滤器 - 移除商品带货动态`. (#4425)
> 移除动态里的商品带货动态 (UP主的推荐 · 来自 XX), 装有 `动态过滤器` 时生效.

- 新增组件 `动态图片平铺展示`. (#4645)
> 将动态中左右切换式的图片改回传统的平铺展示. (在动态详情中可能稍有延迟)

- 更换了 `自定义顶栏` 的动态提醒数字来源. (#4427)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.8.9-preview'>[预览] v2.8.9</a>.</em>

**Comment by kethoven** [comment]
自定义顶栏-稍后再看 视频数量显示错误。
我自己的稍后再看列表有1000个视频，但是他只显示到970.

**Reply by the1812** [reply]
这玩意上限不是 100 吗?

**Reply by Geeyun-JY3** [reply]
能破百好像已经有半年了吧（我怎么印象中之前好像有人在稍后再看能破百后来报过`自定义顶栏`会有相关报错的 bug（也有可能是记错了

**Reply by kethoven** [reply]
现在限制是1000个，上一个版本的数目是正常的。
这个版本开始，数目才出错的。

**Comment by kethoven** [comment]
> 这玩意上限不是 100 吗?



> 这玩意上限不是 100 吗?



> 这玩意上限不是 100 吗?

最近尝试降级，但是好像组件无法降级。问题依然存在。这个问题是稳定存在的一个新bug。
